packages/gatsby-theme-ui-preset/package.json

```diff
@@ -1,11 +1,10 @@
 {
   "name": "gatsby-theme-ui-preset",
   "version": "0.0.31",
-  "description": "A Gatsby theme for theme-ui styles",
-  "main": "index.js",
+  "description": "A preset for theme-ui styles",
+  "main": "src/index.js",
```

Comment:
Should `main` and `module` point to the `dist` directory since it looks like `microbundle` is set up below

Reply:
Technically we're not using microbundle at all in this implementation. See next comment. Going to remove it for now and we can decide if we want to use the other tool.